doc/doc-site/docs/overview/orders.md

```diff
@@ -215,297 +215,214 @@ Econia does not use a critical height of 1. The actual critical height is define
 
-1. Pick a decimal lot size.
+```python
+>>> tick_size_unit_new = "0.000001"
+>>> get_tick_size_integer(tick_size_unit_new, quote_decimals)
+1
+```
 
-   ```python
-   >>> lot_size_decimal = dec('0.1')
-   ```
+Great!
+That just so happens to be the smallest possible tick size in this case.
+Now let's plug this tick size back into the minimum and maximum price formulas to check our work:
 
-1. Assert that integer lot size corresponds to an integer multiple of base subunits.
+```python
+>>> get_min_quote_per_base_nominal(lot_size_unit, tick_size_unit_new)
+0.001
+>>> get_max_quote_per_base_nominal(lot_size_unit, tick_size_unit_new)
+4294967.295
+```
 
-   ```python
-   >>> to_check = lot_size_decimal * 10 ** decimals_base
-   >>> lot_size_integer = int(to_check)
-   >>> assert to_check >= 1 and to_check == lot_size_integer
-   ```
+Notice the maximum price has gone down by one order of magnitude, but it's still "plenty" high.
+We're almost done at this point: we just need a minimum order size.
+Let's require a minimum limit order value of \$50, that is ~6.83 `APT`, equal to $\$50 / \$7.32$.
+Using the SDK:
 
-1. Pick a decimal tick size.
+```python
+>>> get_min_size_integer("6.83", base_decimals, lot_size)
+6830
+```
 
-   ```python
-   >>> tick_size_decimal = dec('0.01')
-   ```
+Above is the minimum number of lots that a limit order must have before it can be placed.
+Remember that market orders can be of any size!
+We are done configuring the market now.
+These are our results:
 
-1. Assert that the integer tick size corresponds to an integer multiple of quote subunits.
+| Parameter    | Units                | Value  | Meaning                  |
+| ------------ | -------------------- | ------ | ------------------------ |
+| Lot size     | Subunits (of `APT`)  | 100000 | Order size granularity   |
+| Tick size    | Subunits (of `USDC`) | 1      | Lot value granularity    |
+| Minimum size | Lots (of `APT`)      | 6830   | Minimum limit order lots |
 
-   ```python
-   >>> to_check = (lot_size_decimal * tick_size_decimal
-   ...             * 10 ** decimals_quote).normalize()
-   >>> tick_size_integer = int(to_check)
-   >>> assert to_check >= 1 and to_check == tick_size_integer
-   ```
+Always remember to check the minimum and maximum price of your configuration to ensure adequate price granularity and room for movement both upwards and downwards!
 
 ### Noteworthy examples
 
 Consider the trading pair `wBTC/USDC`:
-as of the time of this writing, one `BTC` costs approximately 17792.27 `USD`, so initial choices for lot size and tick size might entail:
+as of the time of this writing, one `BTC` costs approximately 17792.27 `USD`, so initial choices for lot size and tick size might be:
 
-1. 0.00001 `wBTC` decimal lot size (corresponding to 0.1779227 `USDC` nominal).
+1. 0.00001 `wBTC` decimal lot size (a value of 0.1779227 `USDC`).
 1. 0.01 `USDC` decimal tick size.
```

Comment:
I think this example as originally written (by me) was confusing, and I suggest taking out the term "decimal tick size"

Also, `wBTC` should be 9 decimals, and given that it isn't mentioned up front I suspect this example needs to be rewritten